README.md

```diff
@@ -73,21 +73,46 @@ assert!(text_result.as_text().starts_with("; SPIR-V\n"));
 
 Setup
 -----
+shaderc-rs needs [shaderc library](https://github.com/google/shaderc) be installed
+
+shaderc library can be obtained from [here](https://github.com/google/shaderc#downloads)
+and linked, or can be [built from source](#building-from-source):
+
+1. Setup, using static linking
```

Comment:
I don't think we want ordered list: it's not indicating some sort of order; it's just different configurations.